README.md

```diff
@@ -19,13 +19,48 @@ Run `nix-build -A tezos-client-static -o tezos-static`
 to build staticically linked `tezos-client` executable.
 
 Run `nix-build -A tezos-client-{rpm, deb}-package -o tezos-client-package` in order
-to build native `.rpm` or `.deb` package for RedHat and Debiab-based distros.
+to build native `.rpm` or `.deb` package for Red Hat and Debian-based distros.
 
 ## Obtain binary or packages from CI
 
 If you don't want to build these files from scratch, you can download artifacts
-produced by the CI. Go to the [master builds](https://buildkite.com/serokell/tezos-client/builds?branch=master)
-and download the artifacts from the `build and package` stage.
+produced by the CI. Go to the [latest master build](https://buildkite.com/serokell/tezos-client/builds/latest?branch=master),
+click on `build and package` stage, choose `Artifacts` section and download files by clcking on the filenames.
+
+## Ubuntu (Debian based distros) usage
+
+### Install downloaded `.deb` package
+
+Download `.deb` file from the CI and double-click on it or run:
+```
+sudo apt install <path to deb file>
+```
+
+### Use PPA with `tezos-client`
+
+Also if you are using Ubuntu you can use PPA in order to install `tezos-client`.
+In order to do that run the following commands:
+```
+sudo add-apt-repository ppa:serokell/tezos && sudo apt-get update
+sudo apt-get install tezos-client
```

Comment:
Have you tested these commands btw (I mean all commands present in this file)?

Reply:
Everything except Fedora (I nuked VM with it in order to free some space for nix :(), but it this package is similar to one, I've been building earlier, so it should work as well

Reply:
UPD: tested on Fedora as well